closed/adds/jdk/src/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -919,34 +752,52 @@ boolean isRestrictedServiceAllowed(Service service) {
                 return false;
             } else if (constraints.length == 0) {
                 // Allow this provider with no constraints.
+                if (debug != null) {
```

Comment:
Please copy the debug code from later versions, before line 752, and add a blank line before line 848.

Reply:
Done.